cyclic_boosting/base.py

```diff
@@ -280,7 +281,7 @@ def _init_default_feature_groups(self, X):
         if self.weight_column is not None:
             exclude_columns.append(self.weight_column)
 
-        self.feature_groups = utils.get_feature_column_names(X, exclude_columns=exclude_columns)
+        self.feature_groups = get_feature_column_names_or_indices(X, exclude_columns=exclude_columns)
```

Comment:
Was surprised to find that there already existed a `get_feature_column_names_or_indices()` function. Is there a specific reason why this function was not used before?

Reply:
Ha, yeah, you are right, we can just use this. An old function I overlooked during refactoring.